aii-ks/src/main/pan/quattor/aii/ks/config.pan

```diff
@@ -322,11 +351,41 @@ variable AII_OSINSTALL_OPTION_FIREWALL ?= null;
 "firewall" ?= AII_OSINSTALL_OPTION_FIREWALL;
 
 
+# Include OS specific kickstart configuration, if needed
+#  - including this at the end allow to undefine tree elements, and remain compatible with other (previous) OSes
+#  - allow 2 types of variants : major and minor OS variants. Variants for major OS version are located in the standard configuration
+#    (as defined by AII_KS_OS_MAJOR_SPECIFIC_INCLUDE, default value should be appropriate when using QWG templates). Variants for minor
+#    OS versions are located into the related OS templates ((as defined by AII_KS_OS_MINOR_SPECIFIC_INCLUDE, default value should be appropriate when using QWG
+#    templates).  When both exist, they are both applied.
+variable  AII_KS_OS_MAJOR_SPECIFIC_INCLUDE ?=
+    if ( is_defined(OS_VERSION_PARAMS['major']) ) {
+        if_exists('quattor/aii/ks/variants/' + OS_VERSION_PARAMS['major']);
+    } else {
+        undef;
+    };
```

Comment:
```suggestion
variable  AII_KS_OS_MAJOR_SPECIFIC_INCLUDE ?= {
    if ( is_defined(OS_VERSION_PARAMS['major']) ) {
        if_exists('quattor/aii/ks/variants/' + OS_VERSION_PARAMS['major']);
    } else {
        undef;
    };
};
```

Reply:
Or: 
```suggestion
variable  AII_KS_OS_MAJOR_SPECIFIC_INCLUDE ?= if ( is_defined(OS_VERSION_PARAMS['major']) ) {
    if_exists('quattor/aii/ks/variants/' + OS_VERSION_PARAMS['major']);
} else {
    undef;
};
```

Reply:
In fact I removed them at the same time you commenting on it.